docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -0,0 +1,176 @@
+
+CAUTION: We advise to use a long validity for your RADIUS server certificate to avoid renew it often.
+
+Client authentication certificate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Generate a certificate from the profile you defined as the client authentication profile, if you generate this certificate manually you will need to use the *download certificate* or *send certificate* buttons. When the certificate is generate via PacketFence, the certificate is automatically integrated into the profile to configure the WiFi network.
+
+Revokation process
+^^^^^^^^^^^^^^^^^^
+
+We have two ways to check if a certificate is valid, you can either use OCSP to interegote the PKI if the certificate is still valid, or use the revokation lists(CRL). However since the OCSP is implemented we advise you to use this method.
+OSCP is scalable, his con would be that one request by certificate is send to the PKI to verify if this certificate is still valid.
+With the CRL everytime the CRL is update every equipement capable of having this CRL has to download it again. For security reason we advise to have a small delay on CRL expiration(to avoid using revoked certificate on the network), but this can generate a lot of network traffic. 
+
+Step3: Configuring PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PacketFence PKI
+^^^^^^^^^^^^^^^
+
+To work with our PKI in PacketFence you will need to add the PKI in the PKI_Provider section.
+
+Click on add a new PKI and select PacketFence PKI.
+
+Follow the instructions below to fill the PacketFence PKI configuration.
+
+image::docs/images/packetfence-pki-provider.png [scaledwidth="100%",alt="PacketFence PKI configuration"]
+
+image::docs/images/packetfence-pki-provider2.png [scaledwidth="100%",alt="PacketFence PKI configuration"]
+
+Certificate storage on PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+With the incomming certificate manager in future version of PacketFence, you should do the following before the installation of certificates.
+
+ # mkdir /usr/local/pf/conf/ssl/tlscert/
+
+Place yourself in the folder where CA are created.
+
+ # cd /usr/local/packetfence-pki/ca/
+ # mv YourCA.pem /usr/local/pf/conf/ssl/tlscert/
+ # chown pf:pf /usr/local/pf/conf/ssl/tlscert/*
+
+Download or send the RADIUS server certificate using the button on the PKI, and copy it into */usr/local/pf/conf/ssl/*. 
+
+For the RADIUS server authentication you will have 2 files, the certificate file and private key, you need both for RADIUS to work properly so make sure you copy both in the folder */usr/local/pf/conf/ssl/*.
+
+When you will download or send the certificate it will arrive in P12 format with the password send by mail, if you don't receive anything you can look around to see if postfix is properly configured and you filled the information for *Mail P12* as intended.
+
```

Comment:
As I said a bit above. You need to explain how to do the FreeRADIUS EAP TLS configuration.